opentracing-web-servlet-filter/src/test/java/io/opentracing/contrib/web/servlet/filter/AbstractJettyTest.java

```diff
@@ -114,6 +119,34 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
         }
     }
 
+    public static class AsyncServlet extends HttpServlet {
+
+        public static int ASYNC_SLEEP_TIME = 3000;
```

Comment:
This denotes duration of an async task using async request. Outside of this servlet I'm using it in an assert for span duration.

I will use lower value here.

Why is a bad idea to depend on a fixed time?